### PR TITLE
eclipse/rdf4j#2101 document use of writable property

### PR DESCRIPTION
**Pull request by abrokenjester**
- some minor editorial changes as well

### CODE REVIEW

File: site/content/documentation/programming/federation/index.md [hunk 1]
```diff
@@ -445,6 +445,23 @@ FedX supports to use resolvable endpoints as federation members. These resolvabl
 
 Note that also hybrid combinations are possible.
 
+### Example 4: Local Federation with writable endpoint (NativeStore):
```
Comment (aschwarte10) [inline site/content/documentation/programming/federation/index.md]: This should be "Example 5"

As best practice I would suggest to also not restrict writability to a local federation with NativeStore (from the documentation perspective) => we should encourage users to rather move to repository manager based solutions and resolvable repositories (which of course then can be NativeStore). This is a more stable environment.

If you are OK, I will provide a commit on top

**Review by aschwarte10 — CHANGES_REQUESTED** [review]
Looks good, I would like to more generalize this example to not point users into the direction that local NativeStore repositories should be used. I will provide a commit on-top, feel free to accept or remove and then merge

Maybe in the future, when we have more complex write-strategies we require to spend a seperate paragraph

**Review by aschwarte10 — APPROVED** [review]

**Comment by aschwarte10** [comment]
I have added another commit on top and tried to generalize this a bit. Please have a look and refine as need, then this change can be merged

**Review by abrokenjester — COMMENTED** [review]
Looks good, thanks @aschwarte10

**Pull request merged**